hw3/02-url.js

```diff
@@ -30,9 +30,75 @@ const server = http.createServer((req, res) => {
     res.write(`<h1>Exercise 02</h1>`);
 
     res.write(`<ul> ${routeResults} </ul>`);
+  } 
+
+  
+  else if (req.url.indexOf('/attributes') >= 0){
+    let routeResults = getRoutes();
+    const table = createTableHTML(req.url);
+
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.write(`<h1>hello world & lorem ipsum</h1>`);
+    res.write(`<ul> ${routeResults} </ul>`);
+    res.write(table);
   }
 
-  // Add your code here
+  else if (req.url.indexOf('/items') >= 0){
+    let routeResults = getRoutes();
+    const table = createTableHTML(req.url);
+
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.write(`<h1>hello world & lorem ipsum</h1>`);
+    res.write(`<ul> ${routeResults} </ul>`);
+    res.write(table);
+  }
+
+  else if (req.url.indexOf('/characters') >= 0){
```

Comment:
Styling - The Airbnb style guide requires a space before the leading brace. 
[Section 19.2](https://github.com/airbnb/javascript#whitespace--before-blocks)